README.md

```diff
@@ -4,6 +4,12 @@ A Vue.js frontend of the Open Prices application, to allow users to add prices.
 
 Works with an Open Prices server, see https://github.com/openfoodfacts/open-prices
 
+## Looking for the Open Prices API ?
```

Comment:
made similar to changes after https://github.com/openfoodfacts/open-prices/pull/1013